ios/sdk/WeexSDK/Sources/Model/WXSDKInstance.m

```diff
@@ -396,12 +396,9 @@ - (void)_renderWithData:(NSData *)contents
     }
 
     self.performance.renderTimeOrigin = CACurrentMediaTime()*1000;
-    self.performance.renderUnixTimeOrigin = [WXUtility getUnixFixTimeMillis];
-    long renderOriginTimePlatform = [self.apmInstance onStage:KEY_PAGE_STAGES_RENDER_ORGIGIN];
-    
-    // pass render origin time to page
     [WXCoreBridge setPageArgument:_instanceId key:@"renderTimeOrigin" value:[NSString stringWithFormat:@"%lld", (long long)([[NSDate date] timeIntervalSince1970] * 1000)]];
-    [WXCoreBridge setPageArgument:_instanceId key:@"renderTimeOriginPlatform" value:[NSString stringWithFormat:@"%lld", (long long)renderOriginTimePlatform]];
+    self.performance.renderUnixTimeOrigin = [WXUtility getUnixFixTimeMillis];
+    [self.apmInstance onStage:KEY_PAGE_STAGES_RENDER_ORGIGIN];
```

Comment:
This breaks previous commit?

Reply:
because this file changes after my commit my pr,and I don't pull incubator-weex again.
but I have update the commit now,this have solved.